tests/mock_callables.py

```diff
@@ -0,0 +1,97 @@
+"""
+mocks.py - Mock callables. Objects that can be called and later inquired as to how they were called.
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.
+"""
+from typing import List, NamedTuple, Dict, Any, Tuple
+
+_Call = NamedTuple("_Call", args=tuple, kwargs=Dict[str, Any])
+
+
+class InstanceOf(object):
+    """
+    Simple wrapper around a class. To be used in conjunction with :class:`CallableMock` instances.
+    Instances of :class:`InstanceOf` compare equal with instances of their wrapped type.
+
+    Examples:
+        >>> InstanceOf(str) == "hello"
+        True
+        >>> InstanceOf(dict) == 7
+        False
+    """
+    __slots__ = ("klass",)
+
+    def __init__(self, klass: type):
+        self.klass = klass
```

Comment:
I take it this is some from of pun?

Reply:
What? No, why would this be a pun? `klass` is to avoid the keyword `class` :D